src/dune_rules/jsoo/jsoo_rules.ml

```diff
@@ -137,6 +137,8 @@ end
 
 let install_jsoo_hint = "opam install js_of_ocaml-compiler"
 
+let install_wasoo_hint = "opam install wasm_of_ocaml-compiler"
```

Comment:
has this been released on opam already ?

Reply:
Not yet. I think we should wait for thinks (`wasm_of_ocaml`, the WebAssembly Garbage Collection proposal, and related tools) to stabilize a bit.

Reply:
The hint is a bit misleading then. Should it say something like
`opam pin add wasm_of_ocaml https://github.com/ocaml-wasm/wasm_of_ocaml.git`
@rgrinberg, any opinion ?